aip/general/0136.md

```diff
@@ -44,6 +44,9 @@ services. The bullets below apply in all three cases.
   - The name **must not** contain prepositions ("for", "with", etc.).
   - The verb in the name **should not** contain any of the standard method verbs ([Get][],
     [List][], [Create][], [Update][], [Delete][]).
+  - The name **must not** include the term `Async`. Instead, the verb `Begin` **may**
```

Comment:
Do we need to suggest anything? The fact that it's an LRO should already indicate it's async, and we should be extrapolating from the RPC interface as much as possible.

I guess the distinction is critical when there's a synchronous / async version. In that case `Start` could also be a good synonym, but that might also cause problems if we're trying to introduce a job pattern. Just brainstorming.

Reply:
If it *is* to differentiate, I'd say we should clarify that you shouldn't add any prefix and the only situation is if you're trying to create an LRO variant.